version

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Can we do this as part of a separate PR?

Probably needs a post publish script that also adds an `unreleased` section if we want to automate this fully.